### PR TITLE
Provide initial german translations for central text blocks

### PR DESCRIPTION
**Pull request by adswa**

### CODE REVIEW

**Comment by tmheunis** [comment]
thank you! I see I had the wrong version of the English text in the German file for the 'about_datacatalog' text. Can you please translate the correct English text from here https://github.com/abcd-j/website/blob/41e7fac2f2a9188781cc147f2cda16521605fae7/i18n/en.yaml#L56-L58

File: i18n/de.yaml [hunk 1]
```diff
@@ -39,23 +39,29 @@
...
+  Daten, erleichtert die Strukturierung von Projekten, gewährleistet
+  Reproduzierbarkeit, unterstützt Kollaboration, und stellt Integrationen zu
+  weit verbreiteter Dateninfrastruktur zur Verfügung."
 
 - id: about_datacatalog
-  translation: "[Pending translation] DataLad Catalog is a free and open source command line tool with a Python API that
-            allows you to turn structured metadata into a user-friendly, browser-based data catalog."
+  translation: "Der DataLad Catalog ist ein frei verfügbares open source
+  Softwarewerkzeug, um strukturierte Metadaten in nutzerfreundliche,
+  browser-basierte Datenkataloge zu verwandeln."
```
Comment (adswa) [inline i18n/de.yaml]:
```suggestion
  translation: "Der Datenkatalog stellt eine Übersicht über einer Vielzahl erhobener Datensätze dar, die in Partnerinstitutionen erhoben und geteilt wurden."
```

**Comment by adswa** [comment]
done :)

**Comment by tmheunis** [comment]
Thanks @adswa!

**Pull request merged**